ios/RCPurchaserInfo+RNPurchases.m

```diff
@@ -8,25 +8,43 @@
 
 #import "RCPurchaserInfo+RNPurchases.h"
 
+static id formatter;
+static dispatch_once_t onceToken;
+
+static NSString * stringFromDate(NSDate *date)
+{
+    dispatch_once(&onceToken, ^{
+        if (@available(iOS 10.0, *)) {
+            formatter = [NSISO8601DateFormatter new];
+        } else {
```

Comment:
Great find! Sorry, I didn't consider the 9.x case when I wrote the Objective-C wrapper for React Native.

One small change though, I don' think we need to ever use the `NSISO8601DateFormatter`, we can just always use the fallback.

It works either way but having the one path simplifies the code.

Reply:
You're right, let's keep it simple.